examples/concertbot/Makefile

```diff
@@ -13,21 +13,19 @@ help:
 	@echo "        Show your stories as a graph."
 
 run:
-	make run-actions&
-	make run-core
+	rasa shell -m models --endpoints endpoints.yml
 
 run-core:
-	python -m rasa.core.run --core models/dialogue --endpoints endpoints.yml
+	rasa run core -m models --endpoints endpoints.yml
```

Comment:
If there is no channel connecter, `rasa run` is falling back to the command line. So basically, `rasa shell` and `rasa run` are the same here.

Reply:
I know they're the same, I guess it was just a question of usage since the `rasa run` command throws this `No chat connector configured, falling back to the command line. Use `rasa configure channel` to connectthe bot to e.g. facebook messenger.` and `rasa shell` doesn't because it's already set to fall back